terraform/production/main.tf

```diff
@@ -22,16 +22,3 @@ terraform {
     }
 }
 
-data "aws_vpc" "production_vpc" {
-    tags = {
-        Name = "vpc-production-apis-production"
-    }
-}
-data "aws_subnet_ids" "production" {
-    vpc_id = data.aws_vpc.production_vpc.id
-    filter {
-        name   = "tag:Type"
-        values = ["private"]
-    }
-}
-
```

Comment:
Why are we removing these?

Reply:
They contain terraform for DMS. Is there a a database for this API?

Reply:
Not sure, if we were going to have one for this API. 
These describe things related to the `VPC` and `subnetIds` so was wondering if this would remove the VPC set up?

Reply:
The VPC settings are used for the DMS task right? There's also no security group attached